src/Parsers/ASTColumnDeclaration.cpp

```diff
@@ -97,6 +102,12 @@ void ASTColumnDeclaration::formatImpl(const FormatSettings & settings, FormatSta
         settings.ostr << ' ' << (settings.hilite ? hilite_keyword : "") << "TTL" << (settings.hilite ? hilite_none : "") << ' ';
         ttl->formatImpl(settings, state, frame);
     }
+
+    if (collation)
+    {
+        settings.ostr << ' ' << (settings.hilite ? hilite_keyword : "") << "COLLATION" << (settings.hilite ? hilite_none : "") << ' ';
```

Comment:
We parse `COLLATE` but format as `COLLATION`
```suggestion
        settings.ostr << ' ' << (settings.hilite ? hilite_keyword : "") << "COLLATE" << (settings.hilite ? hilite_none : "") << ' ';
```